api/api.go

```diff
@@ -337,7 +347,8 @@ func ListEventUsage(db db.SQLClient) echo.HandlerFunc {
 				memory_in_mb,
 				iso8601(lower(duration)) as from,
 				iso8601(upper(duration)) as to,
-				sum(price::bigint) as price
+				sum(price_inc_vat::bigint) as price_inc_vat,
```

Comment:
Clearly you didn't introduce this...but..

[One of these things is not like the other](https://www.youtube.com/watch?v=ueZ6tvqhk8U) ... pretty sure it's a bug/omission that `ListEventUsage` is the only endpoint to return price not "in pence" and just truncating the decimal!

However I think the x100 should generally live in the monitize_resources query (if anywhere) so I'm making a note to clean this up as part of the story that decides what endpoints we really need.... so don't expect this fixed now.

Reply:
Yeah, was wondering about this myself. I'm fine with changing these in the follow-up story.